data processing/file_joiner.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+# Specify the directory path
+directory_path = r'C:\Users\camil\Documents\ScoutingData'
+
+# Get a list of all ".txt" files in the directory
+json_files = [file for file in os.listdir(directory_path) if file.endswith('.json')]
+
+# Initialize an empty array
+merged_content = '['
+
+# Separator starts empty for first file
+separator = ''
+
+# Read the contents of each ".txt" file and append to the merged content
+for json_file in json_files:
+    file_path = os.path.join(directory_path, json_file)
+    with open(file_path, 'r', encoding='utf-8') as in_file:
+        merged_content += separator
+        separator = ','
+        merged_content += in_file.read()
```

Comment:
Better written as:
`merged_content += in_file.read() + ','`
Remove the separator on the last iteration and remove the separator variable.